### PR TITLE
BREAKING CHANGE: align brand name with backend

### PR DESCRIPTION
**Pull request by mmataja**
References 
https://autoricardo.atlassian.net/browse/DM-1283

## Motivation and context
Idea is to align brand names with the ones on the backend side so that we don't have to some unnecessary ugly mappings.
Reason why it's in enum so that in case we have to change it again we change it in one place instead of searching throughout the project where could that change affect something. Hopefully everything is covered :/

## Before
Brand names were `as24` and `ms24`

## After
Brand names are `autoscout24` and `motoscout24`

## How to test
Everything should work as before.

### CODE REVIEW

File: src/components/themeProvider/index.tsx [hunk 1]
```diff
@@ -1,16 +1,18 @@
 import React, { FC, PropsWithChildren } from 'react';
 import { ChakraProvider } from '@chakra-ui/react';
 
+import { Brand } from 'src/types/brand';
+
 import { autoScout24Theme, motoScout24Theme } from 'src/themes';
 
 export type Props = {
   // Theme to use
```
Comment (talamcol) [inline src/components/themeProvider/index.tsx:9]: very useful comment 🦭 🤣

**Review by talamcol — APPROVED** [review]
looks good to me. I'd suggest to include the pre-release of this PR into listings-web and seller-web and let QA test it quickly. 

Don't forget to issue a BREAKING CHANGE, not feat.

File: src/types/brand.ts [hunk 2]
```diff
@@ -1 +1,4 @@
-export type Brand = 'as24' | 'ms24';
+export enum Brand {
+  AutoScout24 = 'autoscout24',
+  MotoScout24 = 'motoscout24',
```
Comment (GutierrezGo) [inline src/types/brand.ts:3]: I'am just wondering if we want to be in sync with what we have in listings project. In listings project we have `Brand.Ms24` `Brand.As24`.

Did you take that into consideration?

File: src/components/themeProvider/index.tsx (hunk 1 above)
Comment (Averethel) [reply src/components/themeProvider/index.tsx:9]: might've been added for docgen

File: src/types/brand.ts (hunk 2 above)
Comment (mmataja) [reply src/types/brand.ts:3]: Yea yea, this will be applied to all projects, listing, seller...Idea is to align everything with BE

**Review by lkappeler — APPROVED** [review]

**Review by Milos-Zbiljic — APPROVED** [review]

**Review by GutierrezGo — APPROVED** [review]

**Pull request merged**